frontend/src/containers/RecipePage/Recipe.js

```diff
@@ -0,0 +1,151 @@
+import Header from '../../components/Header/Header';
+import TagsBar from './TagsBar/TagsBar';
+import recipephoto from '../../assets/bread.jpg';
+import { withRouter } from 'react-router-dom';
+import './Recipe.css';
+
+class Recipe extends Component {
+  constructor() {
+    super();
+    this.state = {
+     
+      directions: ['bake'],
+      data: [],
+      title: 'Easy Creamy Vegan Macaroni and Cheese',
+      snippet: 'Although this does not taste like the the traditional mac n cheese recipes most of us grew up with it will satisfy your comfort food craving while helping you avoid preservatives, dyes, meat, and dairy.',
+      meal: 'Todo',
+      ingredient: ['1 (8 ounce) package elbow macaroni', '1 cup nutritional yeast', '4 cloves garlice'],
+      photos: '',
+      difficulty: 'EASY',
+      duration: '30',
+      servings: '2',
+
+    };
+  }
+
+
+  render() {
+
+
+
+    const { directions, title, ingredient, snippet, difficulty, duration, servings} = this.state;
+
+
+     const mealDifficulty = () => {
+    if (difficulty === 'EASY') {
+      return (
+        <img className="icons" src="https://img.icons8.com/material-rounded/24/000000/low-connection.png" alt="Easy Difficulty Rating" />
+      );
+    }
+    if (difficulty === 'MEDIUM') {
+      return (
+        <img className="icons" src="https://img.icons8.com/material-rounded/24/000000/medium-connection.png" alt="Medium Difficulty Rating" />
+      );
+    }
+    if (difficulty === 'ADVANCED') {
+      return (
+        <img className="icons" src="https://img.icons8.com/material-rounded/24/000000/bar-chart.png" alt="Advanced Difficulty Rating" />
+      );
+    }
+    return undefined; // TODO: This is probably bad. What should this return for invalid difficulty?
+  };
+
+    return (
+      <div className="overall-container">
+        <div className="row">
+          <div className="header">
+            <Header />
+          </div>
+        </div>
+        <div>
+            <button type="button" className="user-edit-button">
+              <img 
+                src="https://img.icons8.com/windows/32/ffa616/edit.png"
+                alt="edit"
+              />
+
+            </button>
+            <button type="button" className="user-edit-button">
+              <img
+                src="https://img.icons8.com/windows/32/ffa616/cancel.png"
+                alt="delete"
+              />
+            </button>
+        </div>
+        <img src={recipephoto} alt="recipephoto" className="recipe-photo" />
+          
+          <div className="specs-bars">
+          <div className="row">
+         
+          <div id="diff-spec-container1" className="spec-element col-sm-4 text-center">
+              <span className="card-meta"> {mealDifficulty()}
+               {difficulty}</span>
+               </div>
+
+             <div id="diff-spec-container2" className="spec-element col-sm-4 text-center">
+             <span className="Bluez">
```

Comment:
What's this `Bluez` class? Can we call it something a little more descriptive? Also, it looks like the meal difficulty logo doesn't have it, so it's still grey.

Reply:
deleted